string request_type
string action
---
string action
